src/react/features/connector/tools/useOpenFolder.js

```diff
@@ -21,13 +22,15 @@ export const useOpenFolder = () => {
       dispatch(setInactive())
       window.nativeFileDialog.showOpenDialog({ properties: ['openDirectory'] }).then((result) => {
         const path = result.filePaths[0]
-        connect(channels.GENERATE_DIOGRAPH, path).then(({ id }) => {
-          console.log('Open folder')
```

Comment:
I would like to have this kind of set of logging in all channels.

Reply:
Logging is again there!